packages/ui5-tooling-modules/lib/utils/WebComponentRegistry.js

```diff
@@ -37,6 +37,7 @@ const calculateGetterName = (functionName) => {
 // the class name of the base class of all control wrappers
 // corresponds to the "sap.ui.core.webc.WebComponent" class at runtime.
 const UI5_ELEMENT_CLASS_NAME = "UI5Element";
+const WEBC_ASSOCIATION_TYPE = "HTMLElement | string | undefined";
```

Comment:
If the order can be guaranteed, I'm ok with this - otherwise I would suggest an array to match. But for a first drop I'm fine